### ISSUE TITLE
Rubinius support

### ISSUE THREAD

**Issue by paddor** [rspec/rspec-core#2241]
What about officially supporting Rubinius?

**Comment by myronmarston** [comment]
To the extent that Rubinius behaves like MRI and/or JRuby, it's supported.  I believe RSpec works on Rubinius but it's not something we claim to officially support because we have not been able to get consistent passing CI builds.  This is something we have invested a ton of time in over the years (If I had to guess, I'd guess that we've invested over 100 hours in it) and we've never been able to get Rubinius to pass consistently.  We reached the conclusion that it's simply not feasible for us to support it.  (In contrast, we've never had similar problems on MRI or JRuby).

However, if someone from the community (perhaps yourself?) wants to step forward and provide support, we'd happily merge PRs that fix Rubinius issues and if someone wanted to commit to that long-term we'd be glad to add the to the core team.
